docker-compose.yml

```diff
@@ -60,7 +60,7 @@ services:
     build:
       context: .
       dockerfile: integrations/python.Dockerfile
-    command: 'npx jest src/integration.test.ts'
+    command: 'npx vitest run src/integration.test.ts'
```

Comment:
Have to use the `run` command here because Vitest doesn't know in this Docker container that it's being run in a CI environment and it'll sit there in watch mode.